cmake/CeleritasUtils.cmake

```diff
@@ -647,6 +647,10 @@ function(celeritas_get_g4libs var)
     foreach(_lib IN LISTS ARGN)
       set(_lib Geant4::G4${_lib}${_ext})
 
+      # Tasking was merged into core management
+      if(_lib MATCHES "tasking" AND NOT TARGET "${_lib}")
+        continue()
+      endif()
```

Comment:
If we call

```
celeritas_get_g4libs(OUTPUT_VAR tasking)
```

then `OUTPUT_VAR` would be empty... The problem here is if `tasking` is requested, then do we want the stuff in `G4global`, or that in `G4run`? I'm guessing that in _most_ cases, it's going to be `G4run`, but I'd have to look at the specifics of each call.

Reply:
Maybe we just silently omit libraries that aren't there, and specify `tasking run` as arguments so that one or the other will get picked up?

Reply:
Yes, that would work, we'll just have to be aware of that should those calls change.

Reply:
Well, how about we do it with the manual library checks and matching, and use the newer (more granular) libraries in the *calls* to this function and map them to the older (coarser) `persistency` library.

Reply:
I think for now it's probably better to stick with the existing names _for now_. We could review this depending on what is done in Geant4 11.2.2/11.3 with bug fixes in the former and what granularity is arrived at in 11.3.